planner/mapper/targetable.go

```diff
@@ -86,6 +86,51 @@ func NewFilter() *Filter {
 	}
 }
 
+func (f *Filter) ToMap(mapping *core.DocumentMapping) map[string]any {
+	return filterObjectToMap(mapping, f.Conditions)
+}
+
+func filterObjectToMap(mapping *core.DocumentMapping, obj map[connor.FilterKey]any) map[string]any {
+	outmap := make(map[string]any)
+	if obj == nil {
+		return nil
+	}
+	for k, v := range obj {
+		switch keyType := k.(type) {
+		case *PropertyIndex:
+			subObj := v.(map[connor.FilterKey]any)
+			outkey, _ := mapping.TryToFindNameFromIndex(keyType.Index)
+			if childMapping, ok := tryGetChildMapping(mapping, keyType.Index); !ok {
+				outmap[outkey] = filterObjectToMap(mapping, subObj)
+			} else {
+				outmap[outkey] = filterObjectToMap(childMapping, subObj)
+			}
+
+		case *Operator:
+			switch keyType.Operation {
+			case "_and", "_or":
```

Comment:
suggestion: `"_and"` and `"_or"` should really be consts in the `client.request` package.  Marked as suggestion as you could happily argue that this is out of scope and you are just moving them.

Reply:
Theres a number of spots this could be applied, so Ill mark as out of scope, and open an issue 👍